src/services/users/auth.js

```diff
@@ -0,0 +1,21 @@
+import { users } from "../../mock/users";
+
+export const auth = (email, password) => {
+  const response = users.filter((user) => user.email === email)[0];
```

Comment:
Aqui também poderia ser um find bunito

```suggestion
  const response = users.find((user) => user.email === email);
```